synapse/models/inet.py

```diff
@@ -2015,8 +2015,11 @@ def getModelDefs(self):
                             'doc': 'The guid of the destination process.'
                         }),
                         ('dst:exe', ('file:bytes', {}), {
-                            'doc': 'The file (executable) that received the connection.'
-                        }),
+                            'doc': 'The file (executable) that received the connection.'}),
+
+                        ('dst:txfiles', ('array', {'type': 'file:attachment', 'sorted': True, 'uniq': True}), {
```

Comment:
Does it make sense to have these sorted since they're guid forms? I'm specifically thinking about any overhead associated with the sorting.

Reply:
Yes. Sorting makes it so that reassigning the same set of things does not result in a write.